website/static/js/pages/dashboard-page.js

```diff
@@ -10,29 +9,15 @@ var ko = require('knockout');
 var $ = require('jquery');
 var jstz = require('jstimezonedetect').jstz;
 
-// Knockout components for the onboarder
-require('js/onboarder.js');
 var $osf = require('js/osfHelpers');
+var FileBrowser = require('js/file-browser.js');
 var LogFeed = require('js/logFeed');
-var ProjectOrganizer = require('js/projectorganizer').ProjectOrganizer;
+var m = require('mithril'); // exposes mithril methods, useful for redraw etc.
 
 var url = '/api/v1/dashboard/get_nodes/';
 var request = $.getJSON(url, function(response) {
     var allNodes = response.nodes;
-    //  For uploads, only show nodes for which user has write or admin permissions
-    var uploadSelection = ko.utils.arrayFilter(allNodes, function(node) {
-        return $.inArray(node.permissions, ['write', 'admin']) !== -1;
-    });
-
-    // If we need to change what nodes can be registered, filter here
-    var registrationSelection = ko.utils.arrayFilter(allNodes, function(node) {
-        return $.inArray(node.permissions, ['admin']) !== -1;
-    });
-
-    $osf.applyBindings({nodes: allNodes}, '#obGoToProject');
-    $osf.applyBindings({nodes: registrationSelection, enableComponents: true}, '#obRegisterProject');
-    $osf.applyBindings({nodes: uploadSelection}, '#obUploader');
-    $osf.applyBindings({nodes: allNodes}, '#obCreateProject');
+    //.. Getting nodes
```

Comment:
No, I thought at some point I might need that call but since the switch to V2 that's not necessary